### ISSUE TITLE
Expanded text not announced by NVDA, unless it is a list.

### ISSUE THREAD

**Issue by hansgrimm** [nvaccess/nvda#8101]
### Steps to reproduce:
1. open Internet Explorer 11 on Win 8.1
2. browse to https://www.spoordeelwinkel.nl/samenstellen/uitjes/keukenhof-2018.html
3. tab to the FAQ section ("Veelgestelde vragen")
4. expand the item. arrow down to have the item read.
5. text-only items not read, lists are read

### Expected behavior:
All expanded items should be read

### Actual behavior:
If the expanded item is text-only, it is skipped. It is only read if it contains a list.

### System configuration:
NVDA version: 2018.1

NVDA Installed or portable: installed

Windows version: Win 8.2 build 9600

Name and version of other software in use when reproducing the issue: Internet Explorer 11

### Other questions:

Does the issue still occur after restarting your PC?
Yes

Have you tried any other versions of NVDA?
No, but the issue persists across updates.

**Comment by bramd** [comment]
The text is readable using object navigation. Using nvda+f1 shows that the problematic text seems to be a text node, where the readable text is a p. Since the unreadable text is just a p in a section, I'm unsure why it is not recognized as a p.

**Comment by hansgrimm** [comment]
Thanks @bramd Maybe we can fix the issue at our end!

**Comment by bramd** [comment]
Probably, but it would be nice if NVDA can handle this correctly in IE 
as well (it works fine in Chrome/Firefox). Could you create a minimal 
test document to reproduce this issue? Just a ul, li, section, p 
structure didn't trigger this bug here. A minimal test case would help 
identifying the issue and allows for testing even if you change the website.

**Comment by hansgrimm** [comment]
I’ll see what I can do!

**Comment by Adriani90** [comment]
@hansgrimm is this meanwhile working in NVDA 2019.1.1? The link in the description says "page not found". Do you have a minimal test case where we can reproduce this issue?

**Comment by hansgrimm** [comment]
> @hansgrimm is this meanwhile working in NVDA 2019.1.1? The link in the description says "page not found". Do you have a minimal test case where we can reproduce this issue?

I am no longer working on this project, so I do not know the current status. The URL no longer works because the site has moved, that much I do know ;)

I will refer this question to my former teammates. Thanks for your help!

**Comment by Adriani90** [comment]
It seems we are not able to reproduce this issue with the current description since the link does not work. If you have any updates, please comment on this issue and we can reopen it. Thanks.
